.gitignore

```diff
@@ -1,11 +1,13 @@
 project
 smarty/templates_c
+.gitignore
```

Comment:
I don't think these should be ignored. This file and the `.github/` folder are both tracked in the repo.

Reply:
ooops. will be corrected in the next commit.